### PR TITLE
task/frontend46 add how long ago created helper 

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
[Create helper to convert date to string 'how long ago was created'.](https://trello.com/c/n25254Mv/46-create-helper-to-convert-date-to-string-how-long-ago-was-created)

File: frontend/package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "@hookform/resolvers": "2.9.6",
     "@reduxjs/toolkit": "1.8.3",
     "clsx": "1.2.1",
+    "dayjs": "^1.11.4",
```
Comment (VitaliiBedletskyi) [inline frontend/package.json]: Do not use ^ operator for specifying dependency version

File: frontend/src/helpers/date/get-how-long-ago-string/get-how-long-ago-string.helper.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import dayjs from 'dayjs';
+
+type timeUnitType = 'year' | 'month' | 'week' | 'day' | 'hour' | 'minute';
+const timeUnitArray: timeUnitType[] = ['year', 'month', 'week', 'day', 'hour', 'minute'];
+
+const getHowLongAgoString = (date: Date): string => {
+  const inputDate = dayjs(date);
+  const nowDate = dayjs();
+  for (const timeUnit of timeUnitArray) {
```
Comment (VitaliiBedletskyi) [inline frontend/src/helpers/date/get-how-long-ago-string/get-how-long-ago-string.helper.ts]: Can be simplify like: 
```
const moreThanMinute = timeUnitArray.find(timeUnit => nowDate.diff(inputDate, timeUnit) > 0);

if (moreThanMinute) {
  return `${different} ${timeUnit} ago`;
}

return 'less than minute ago';
```

**Review by VitaliiBedletskyi — CHANGES_REQUESTED** [review]

File: frontend/src/helpers/date/get-how-long-ago-string/get-how-long-ago-string.helper.ts (hunk 2 above)
Comment (ghost) [reply frontend/src/helpers/date/get-how-long-ago-string/get-how-long-ago-string.helper.ts]: done

File: frontend/package.json (hunk 1 above)
Comment (ghost) [reply frontend/package.json]: done

**Review by VitaliiBedletskyi — APPROVED** [review]

**Pull request merged**